docs/ELRS_RECEIVERS.md

```diff
@@ -0,0 +1,63 @@
+# mLRS Documentation: ELRS Receivers #
+
+([back to main page](../README.md))
+
+## Supported ELRS Receivers ##
+
+As of April 2024, ELRS receivers with an ESP8285 MCU are experimentally supported by mLRS. The following targets are supported:
+
+- Generic 900
+- Generic 900 PA
+- Generic 2400
+- Generic 2400 PA
+
+To determine if your receiver hardware is supported, go to [ELRS Targets](https://github.com/ExpressLRS/targets/blob/master/targets.json) and find the layout file of your hardware.
+
+Note: ELRS receivers that use an ESP32 MCU are under active development.
```

Comment:
Note is bold and italic on teh other page. What's the style guide here?

Reply:
Note should have no extra formatting (per other pages), so have removed.